### PR TITLE
GRW-1641 / Fix / Faster codegen

### PR DESCRIPTION
**Pull request by alebedev**
## Describe your changes

- Remove prettier and eslint steps from
- Simplify turbo config by lifting up common settings

Codegen time on my machine (median of 3 runs)
before: 9.1s
after: 3.8s 

## Justify why they are needed

Better developer experience

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Review by gustaveen — APPROVED** [review]
Nice if this works! 🙌 
Do we know the reason why we ran prettier and lint in codegen, did we need to format it?

File: turbo.json [hunk 1]
```diff
@@ -1,63 +1,45 @@
...
-      "outputs": [".next/**"],
-      "dependsOn": ["^build", "codegen"]
-    },
-    "onboarding#test": {
-      "outputs": ["coverage/**"],
-      "dependsOn": ["^build"]
+    "codegen": {
+      "inputs": ["src/graphql/*.graphql", "codegen.yml"],
+      "outputs": ["src/services/**/generated.ts", "src/services/apollo/types.ts"],
+      "cache": false
```
Comment (robinandeer) [inline turbo.json:7]: Hmm just wondering what this means? Does Turbo do anything with inputs/outputs if we turn off the cache?

**Comment by alebedev** [comment]
I assume the reason we ran prettier and lint on generated files was that it broke.  I find it's better to exclude generated files from check - we're not linting anything in "dist", for example

File: apps/onboarding/package.json [hunk 2]
```diff
@@ -13,7 +13,7 @@
     "lint:fix": "next lint --fix --ignore-path .gitignore",
     "clean": "rm -rf .turbo && rm -rf node_modules && rm -rf .next",
     "download-translations": "./scripts/download-translations.sh",
-    "codegen": "graphql-codegen && prettier --write ./src/services/**/{generated,types}.ts && yarn lint:fix",
+    "codegen": "graphql-codegen",
```
Comment (robinandeer) [inline apps/onboarding/package.json:16]: We don't need to load "dotenv" here?

**Review by robinandeer — COMMENTED** [review]
Great stuff! Just asking stuff to better understand

File: apps/onboarding/package.json (hunk 2 above)
Comment (alebedev) [reply apps/onboarding/package.json:16]: It's hardcoded, don't want to change it now

**Pull request merged**